src/go/k8s/Makefile

```diff
@@ -6,7 +6,7 @@ CONFIGURATOR_IMG_LATEST ?= "localhost/configurator:dev"
 CRD_OPTIONS ?= "crd:trivialVersions=true,preserveUnknownFields=false"
 
 # default redpanda image to load
-REDPANDA_IMG ?= "localhost/redpanda:dev"
+REDPANDA_IMG ?= "localhost/redpand-operator:dev"
```

Comment:
@RafalKorepta wanted to discuss this, as the make file makes little sense to me here

Reply:
Ok, what is the question?

Reply:
When you build it usually builds the redpanda-operator, but the kustomize is set up to use the repanda:dev image, which makes no sense, was this intentional or is this an error?

Reply:
REDPANDA_IMG is a placeholder for Redpanda container not operator. This is used in kuttle tests everywhere. It should point to the tip of the current branch that test run on. It should be used in the kind load docker-image only.

I personally don't use Makefile to load image to my cluster.

I pull latest nightly build from docker hub and I'm running kuttle test locally after retagging it to `redpanda:dev`